src/libraries/System.Private.CoreLib/src/System/Reflection/AmbiguousMatchException.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Reflection
     public sealed class AmbiguousMatchException : SystemException
     {
         public AmbiguousMatchException()
-            : base(SR.Arg_AmbiguousMatchException)
+            : base(SR.Arg_AmbiguousMatchException_NullMessage)
```

Comment:
Renamed in the resx to match the AmbigiousInvocationException_NullMessage` pattern.